README.md

```diff
@@ -36,16 +36,16 @@ Here is a detailed explanation of each table and its content.
 Has the dishes. Each dish should only appear once in here. Has a one to many relationship with entires in dishes_menus.
 1. id: dish id. Auto-incremented.
 2. name: dish name.
-3. star: 0-3. 0 for no star. 1 for \*. 2 for \*\*. 3 for \*/\*\*.
+3. star: 0-3. 0 for no star. 1 for \*, meaning it must appear in dinner. 2 for \*\*, meaning it must appear in supper. 3 for \*/\*\*, meaning it must appear in both dinner and supper.
```

Comment:
Ok, this is a step closer, but definitely seems to be implying that these dishes ONLY appear for supper or dinner menus. I still think it would be a good idea to mention somewhere that these stars indicate inclusion in default/special meal options, not the meal itself (i.e. a 1 star dish does not only appear at dinner)